src/borg/archive.py

```diff
@@ -146,7 +146,7 @@ def osize_fmt(self):
     def usize_fmt(self):
         return format_file_size(self.usize, iec=self.iec)
 
-    def show_progress(self, item=None, final=False, stream=None, dt=None):
+    def show_progress(self, item=None, final=False, stream=sys.stderr, dt=None):
```

Comment:
This would determine the default (the value of sys.stderr) at import time, while the original code determines it later, when the code is actually called. IIRC, that could be important.

So, to not risk any breakage, just keep the `stream=None` default and add a first line inside the method:
```
stream = stream or sys.stderr
```
And then use stream everywhere in the function where appropriate.